src/transformers/__init__.py

```diff
@@ -486,6 +493,19 @@
     "utils.bitsandbytes": [],
 }
 
+# Beautiful Soup-specific objects
+try:
+    if not is_bs4_available():
```

Comment:
Let's not add bs4 dummies just for one model. For now, we can just have dynamic checks in the modeling file.